ex36.py

```diff
@@ -174,6 +174,17 @@ def lion(current_count):
 	print """
```

Comment:
4 spaces per indent, if possible, please. The god PEP8 will become vengeful. ;-)